x-pack/plugins/monitoring/public/services/executor_provider.js

```diff
@@ -84,9 +85,9 @@ export function executorProvider(Promise, $timeout, timefilter) {
      */
   function start() {
     timefilter.on('fetch', reFetch);
```

Comment:
Just like the almost exact duplicate of this method in Kibana, these listeners need to call their handlers within digest cycles.